cypress/support/step-definitions/form-steps.js

```diff
@@ -0,0 +1,75 @@
+import {
+  saveButton, cancelButton, buttons, additionalActions, leftAlignedActions, rightAlignedActions,
+  errorsSummary, errorMessage, inputValidation,
+} from '../../locators/form';
+
+const ALIGN_PREFIX = 'carbon-form__buttons--';
+
+Then('save button is visible', () => {
+  saveButton().should('be.visible');
+});
+
+Then('save button is not visible', () => {
+  saveButton().should('not.exist');
+});
+
+Then('cancel button is visible', () => {
+  cancelButton().should('be.visible');
+});
+
+Then('cancel button is not visible', () => {
+  cancelButton().should('not.exist');
+});
+
+Then('buttons are aligned to {string}', (direction) => {
+  buttons().should('have.class', ALIGN_PREFIX + direction);
```

Comment:
maybe we should use more readable concatenation form `${ALIGN_PREFIX}${direction}` -> no + here ;)